XenModel/Actions/MultipleAction.cs

```diff
@@ -127,15 +143,26 @@ private void SubActionChanged(ActionBase sender)
             {
                 SubActionTitle = subAction.Title;
                 SubActionDescription = subAction.Description;
+                RecalculatePercentComplete();
                 OnChanged();
             }
         }
 
+        protected virtual void RecalculatePercentComplete()

```

Comment:
If we do that, then we keep incrementing with the percentage of the sender every time the sender changes, which is not correct, as an action's percentage should only be added once.